projects/wave-core-new/src/lib/datatable/table/table.component.ts

```diff
@@ -142,6 +148,70 @@ export class DataTableComponent implements OnInit, AfterViewInit, OnDestroy, OnC
         return feature.getId() === this.projectService.getSelectedFeature().feature;
     }
 
+    coordinateFromGeometry(geometry: OlFeature): string { // For truncated coordinate view in table
+        const coords: string[][] = this.readCoordinates(geometry);
+        const contd: string = (coords[0].length > 1 ? '...' : '');
+        const output: string = ` ${coords[0][0]}, ${coords[1][0]} ${contd}`;
+        return output;
+    }
+
+    /**
+     * Extracts the coordinates of an open layers feature as strings
+     * @param geometry The feature to extract coordinates from
+     * @returns A nested string[][] where index 0 of the outer array are x-Coordinates, index 1 are y-Coordinates
+     */
+    readCoordinates(geometry: OlFeature): string[][] {
+        let xCoords: string[] = [];
+        let yCoords: string[] = [];
+        const type: String = geometry.getGeometry()?.getType();
+        switch (type) {
+            case "Polygon":
+            case "MultiPolygon":
+            case "LineString":
+            case "MultiLineString":
+                const poly: OlPolygon = <OlPolygon>geometry.getGeometry();
+                const l = poly.getCoordinates().length;
+                let allCoords: string[] = [];
+                for (let i = 0; i < l; i++) {
+                    const coord = poly.getCoordinates()[i].toString().split(',');
+                    allCoords = allCoords.concat(coord);
+                }
+                for (let i = 0; i < allCoords.length - 1; i += 2) {
+                    xCoords.push(allCoords[i]);
+                    yCoords.push(allCoords[i + 1]);
+                }
+                break;
+            case "Point": // Works with above (more complicated) method as well ... refactor?
```

Comment:
Did you try it out?

Reply:
I refactored the method so that known types get treated the same (like a polygon, they work the same for the purpose of extracting sets of coordinates) and all other types (besides Polygon, MultiPolygon, LineString, MultiLineString and Point, which I tried out) have "N/A" displayed. It should be easy to include new types if they come up in the future.